helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -503,8 +514,8 @@ private boolean canCompleteSwap(String clusterName, String swapOutInstanceName,
     if (swapInLiveInstance == null) {
       logger.warn(
           "SwapOutInstance {} is {} + {} and SwapInInstance {} is OFFLINE + {} for cluster {}. Swap will"
-              + " not complete unless SwapInInstance instance is ONLINE.",
-          swapOutInstanceName, swapOutLiveInstance != null ? "ONLINE" : "OFFLINE",
+              + " not complete unless SwapInInstance instance is ONLINE.", swapOutInstanceName,
```

Comment:
Suggest not to do entire file reformat, hard for reviewing if it is format change or logical change.